arraySort.js

```diff
@@ -1,31 +1,35 @@
-var evenNumbers = new Array();
-var oddNumbers = new Array();
-var characters = new Array();
+function arraySort(myArray) {
+    var evenNumbers = new Array();
+    var oddNumbers = new Array();
+    var characters = new Array();
 
-let numbers = {
-    evens: evenNumbers,
-    odds: oddNumbers,
-    chars: characters,
-}
+    let numbers = {
+        evens: evenNumbers,
+        odds: oddNumbers,
+        chars: characters,
+    }
 
-function arraySort(myArray) {
-    for (var i = 0; i < myArray.length; i++) {
-        if (isNaN(myArray[i])) {
-            characters.push(myArray[i]);
-        }
-        if (myArray[i] % 2 == 0) {
-            evenNumbers.push(myArray[i]);
-        }
-        if (myArray[i] % 2 == 1) {
-            oddNumbers.push(myArray[i]);
+    if (!Array.isArray(myArray)) {
+        //used return since (throw new Error('invalid input'))error is not received by the test
+        return 'invalid input';
```

Comment:
Throw the error and modify the tests.
You will pass in an invalid value and use `toThrow` or `toThrowErrorMatchingSnapshot` matchers as seen[ here](https://jestjs.io/docs/en/expect#tothrowerror)
For example 
```js
// instead of 
test('It does not accept numbers as input', () => {
    expect(arraySort(5)).toBe('invalid input');
});

// we can do
test('It does not accept numbers as input', () => {
    expect(()=>{arraySort(5)}).toThrow();
});

// or we can do 
test('It does not accept numbers as input', () => {
    expect(()=>{arraySort(5)}).toThrowError('invalid input');
});

// or we can make use of snapshot matching
test('It does not accept numbers as input', () => {
    expect(()=>{arraySort(5)}).toThrowErrorMatchingSnapshot();
});
```